src/search/indexing/local_import.rs

```diff
@@ -73,6 +72,16 @@ pub async fn index_local(
 
                     let project_id: crate::models::projects::ProjectId = ProjectId(m.id).into();
 
+                    let license = match m.license.split(" ").next() {
```

Comment:
wouldn't this break if license ids had spaces in them? or is this not possible in a spdx license id

Reply:
or, shouldn't this parse the entire spdx expression and see if it is osi approved? like if someone's code is lgpl but assets are arr, the filter shouldn't be open source

Reply:
This part is not used for getting the open source filter but for the license filter

Reply:
This just obtains the first license if there are spaces and the full thing otherwise. A space is what separates multiple licenses, but if there are no spaces it's just a single license.